spec/rubocop/cop/style/sole_nested_conditional_spec.rb

```diff
@@ -266,6 +266,82 @@
     RUBY
   end
 
+  context 'when the inner condition has a send node without parens' do
+    context 'in guard style' do
+      it 'registers an offense and corrects' do
+        expect_offense(<<~RUBY)
+          if foo
+            do_something if ok? bar
+                         ^^ Consider merging nested conditions into outer `if` conditions.
+          end
+        RUBY
+
+        expect_correction(<<~RUBY)
+          if foo && (ok? bar)
```

Comment:
A little worrisome is that `foo && ok?(bar)` is (maybe) more natural code rather than `foo && (ok? bar)` for user 😅

Reply:
Yeah, I agree, but I couldn't get it working otherwise. If I try to correct the `condition` node I get a clobbering error. Any suggestions?

Reply:
Unfortunately I don't have an immediate idea 💦 , but I think it can be improved later. First, I'm going to merge this PR to prevent the syntax error. Thank you for your consideration and fixing!

Reply:
Thanks @koic!